docs/core/whats-new/dotnet-core-2-1.md

```diff
@@ -43,7 +43,9 @@ A number of tools that were available only on a per project basis using [`Dotnet
    ```console
    dotnet watch -- --verbose build
    ```
-
+  
+   In this example, the `--` option that precedes the `--verbose` option. It delimits the options passed directly to the `dotnet watch` command from the arguments that are passed to the child `dotnet` process. Without it, the `--verbose` option applies to the `dotnet watch` command, not the `dotnet build` command.
+  
```

Comment:
Feels like a grammar problem in the first sentence

Reply:
It is a grammar problem, @KathleenDollard. Thanks for pointing it out; I've fixed it.